content/enterprise_influxdb/v1.9/administration/configuration.md

```diff
@@ -142,3 +157,24 @@ relevant position number (in this case: `0`):
 For the Nth Graphite configuration in the configuration file, the relevant
 environment variables would be of the form `INFLUXDB_GRAPHITE_(N-1)_BATCH_PENDING`.
 For each section of the configuration file the numbering restarts at zero.
+
+### `GOMAXPROCS` environment variable
+
+> ***Note:*** `GOMAXPROCS` cannot be set using the InfluxDB configuration file.
+> It can only be set as an environment variable.
+
```

Comment:
```suggestion

{{% note %}}
_**Note:**_ `GOMAXPROCS` cannot be set using the InfluxDB configuration file.
It can only be set as an environment variable.
{{% /note %}}

```